libraries/wavemap/include/wavemap/utils/bits/bit_operations.h

```diff
@@ -1,5 +1,5 @@
-#ifndef WAVEMAP_UTILS_BIT_MANIPULATION_H_
-#define WAVEMAP_UTILS_BIT_MANIPULATION_H_
+#ifndef WAVEMAP_UTILS_BITS_BIT_OPERATIONS_H_
```

Comment:
Technically #pragma once is fine to use despite being non-standard as all sane compilers support it nowadays.